spec/controllers/admin/admin_users_controller_spec.rb

```diff
@@ -476,6 +476,37 @@ def expect_changes_made_by(admin, log_items)
     end
   end
 
+  describe "POST #destroy_user_creations" do
+    let(:admin) { create(:admin) }
+    let(:user) { create(:user, banned: true) }
+    let(:other_owner) { create(:user, banned: false) }
+    let!(:collection1) { create(:collection) }
+    let!(:collection2) { create(:collection) }
+
+    before do
+      collection1.collection_participants = [create(:collection_participant, user: user, collection: collection1, participant_role: CollectionParticipant::OWNER)]
+      collection1.save
+      # Create the second user and make them an owner of the collection
+      create(:collection_participant, user: other_owner, collection: collection2, participant_role: CollectionParticipant::OWNER)
+      create(:collection_participant, user: user, collection: collection2, participant_role: CollectionParticipant::MEMBER)
+    end
+
+    context "when deleting the first user" do
+      it "deletes the first user's collection but preserves the second user's collection" do
```

Comment:
How about `it "only deletes the user's sole owned collections"`?

Reply:
I suppose I could change it, I just wanted to make it clear in the "it" statement the second collection would remain.